tests/conftest.py

```diff
@@ -533,22 +517,62 @@ def subject_v(app, subject_type):
     return vocab
 
 
+@pytest.fixture(scope="module")
+def affiliations_v(app):
+    """Affiliations vocabulary record."""
+    affiliations_service = (
+        current_service_registry.get("rdm-affiliations")
+    )
+    aff = affiliations_service.create(system_identity, {
+        "id": "cern",
+        "name": "CERN",
+        "acronym": "CERN",
+        "identifiers": [{
```

Comment:
I think `identifiers` and `acronym` should not be there ?

Reply:
Yes because affiliations are full records, with a structure a bit different from a generic vocabulary, and in this case we are creating it, then in the `full_records` we only set the `id`.

Reply:
I see, makes sense, thanks!